### PR TITLE
Added softsign activation function and test to PaddlePaddle frontend

### LINKED ISSUE #16245: softsign

### PR DESCRIPTION
**Pull request by sachelsout**
Close #16245

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -53,3 +53,14 @@ def hardtanh(
     less = ivy.where(ivy.less(x, min), min, x)
     ret = ivy.where(ivy.greater(x, max), max, less).astype(x.dtype)
     return ret
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
```
Comment (ZiadAmerr) [inline ivy/functional/frontends/paddle/nn/functional/activation.py:58]: Can we please check whether these are actually the only invalid dtypes for this function?

**Review by ZiadAmerr — COMMENTED** [review]
Amazing work @sachelsout, just this minor check and we are ready to merge.

File: ivy/functional/frontends/paddle/nn/functional/activation.py (hunk 1 above)
Comment (sachelsout) [reply ivy/functional/frontends/paddle/nn/functional/activation.py:58]: image
Hi @ZiadAmerr , I referred [paddle official docs](https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/softsign_en.html) and there it's mentioned, only these 2 dtypes are valid for softsign.

**Comment by ZiadAmerr** [comment]
Hey, @sachelsout please resolve the conflicts and we are ready to merge :)

**Comment by sachelsout** [comment]
> Hey, @sachelsout please resolve the conflicts and we are ready to merge :)

Hello @ZiadAmerr , I resolved the merge conflicts. :)

**Comment by sachelsout** [comment]
> > Hey, @sachelsout please resolve the conflicts and we are ready to merge :)
> 
> Hello @ZiadAmerr , I resolved the merge conflicts. :)

Hi @ZiadAmerr , I resolved the conflicts, but now 1 test is failing. I did not change the code related to softsign. Then how did 1 test failed? Earlier, all the test cases were passing. Am I missing something ?

**Comment by ZiadAmerr** [comment]
Don't worry about it, they are not related to your function. Are the tests passing locally?

**Comment by sachelsout** [comment]
> Don't worry about it, they are not related to your function. Are the tests passing locally?

Yes, the tests are passing locally.

**Comment by ZiadAmerr** [comment]
They ran locally on my end too. LGTM, thanks a lot @sachelsout :)

**Comment by sachelsout** [comment]
> They ran locally on my end too. LGTM, thanks a lot @sachelsout :)

Thank you for reviewing and merging the code !! 😄

**Pull request merged**